### PR TITLE
Update to latest upstream Asciidoctor versions

### PR DESCRIPTION
**Pull request by buuhsmead**
#### What is this PR About?
Describe the contents of the PR
Upstream has released new versions. Would like to sync with those versions.
Removed 'dnf history undo last -y' statement. It removes to much, for instance pip3 binary.

#### How do we test this?
Provide commands/steps to test this PR.
Current provided test in workflow are good enough.

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

File: utilities/ubi8-asciidoctor/Dockerfile [hunk 1]
```diff
@@ -21,37 +22,27 @@ ENV ASCIIDOCTOR_VERSION=${asciidoctor_version} \
...
-  ruby \
-  && dnf clean all && rm -rf /var/lib/dnf && rm -rf /usr/share/doc
-
-# install pandoc
-RUN curl -L https://github.com/jgm/pandoc/releases/download/${PANDOC_VERSION}/pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz | tar xvz --strip-components 1 -C /usr/local/
 
 # Installing Ruby Gems needed in the image
 # including asciidoctor itself
 # TODO: have to find Lasem dev libs     "asciidoctor-mathematical:${ASCIIDOCTOR_MATHEMATICAL_VERSION}" \
+# TODO: have to find a way to get graphviz and java working
```
Comment (garethahealy) [inline utilities/ubi8-asciidoctor/Dockerfile:35]: nit: maybe expand on what this comment means so someone else can pick up

Comment (buuhsmead) [reply utilities/ubi8-asciidoctor/Dockerfile:35]: Lesson learned, do not add TODO as a reminder to myself for further maybe enhancements. Create a separate ticket for those enhancements.
Graphviz is not available in the default ubi8 repositories. So it has to be compiled from source. That compiling of source is the reason for the TODO.

**Review by garethahealy — APPROVED** [review]

**Pull request merged**